shap/_explanation.py

```diff
@@ -826,33 +826,39 @@ def _compute_shape(x) -> tuple[Union[int, None], ...]:
 
 
 class Cohorts:
-    def __init__(self, **kwargs):
+    def __init__(self, **kwargs) -> None:
```

Comment:
This might be a good opportunity to add a class docstring to explain what a cohort represents and how the class is intended to be used.

We don't need to go overboard, but a few docstrings might help future devs get up to speed faster.

Reply:
PS, I think this `kwargs` could be type hinted as:

```suggestion
    def __init__(self, **kwargs: Explanation) -> None:
```

Using syntax explained here:
https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
Thanks, added docstring with examples, please have a read and see if it makes sense.

Reply:
Looks great!